frontend/src/features/users/UserAuthForm/mapping.ts

```diff
@@ -0,0 +1,22 @@
+export type PageType = "sign-in" | "sign-up";
+
+type Title = "ログイン" | "新規登録";
+type PageLink = "/signin" | "/signup";
+type ButtonLabel = "ログインする" | "アカウントを作成する";
+
+export const typeToTitle: Record<PageType, Title> = {
+  "sign-in": "ログイン",
+  "sign-up": "新規登録",
+};
+export const typeToButtonLabel: Record<PageType, ButtonLabel> = {
+  "sign-in": "ログインする",
+  "sign-up": "アカウントを作成する",
+};
+export const typeToLink: Record<PageType, PageLink> = {
+  "sign-in": "/signup",
+  "sign-up": "/signin",
+};
+export const typeToNavigationText: Record<PageType, string> = {
+  "sign-in": "アカウントをお持ちでない方はこちら",
+  "sign-up": "アカウントをお持ちの方はこちら",
+};
```

Comment:
[NITS]
この書き方どう？あんまりRecordを使ったことなくて。

```suggestion
export type PageType = "sign-in" | "sign-up";

export const typeToTitle = {
  "sign-in": "ログイン",
  "sign-up": "新規登録",
} as const;

export const typeToButtonLabel = {
  "sign-in": "ログインする",
  "sign-up": "アカウントを作成する",
} as const;

export const typeToLink = {
  "sign-in": "/signup",
  "sign-up": "/signin",
} as const;
export const typeToNavigationText = {
  "sign-in": "アカウントをお持ちでない方はこちら",
  "sign-up": "アカウントをお持ちの方はこちら",
} as const;

```

Reply:
そんなにこだわりないからRecord使わない形にしといた！https://github.com/givery-bootcamp/dena-2024-team1/pull/156/commits/70c94c6990e60ff8c34d87aa5ee43f4ac0b15cfa